docs/running.rst

```diff
@@ -14,10 +14,23 @@ the site.
 
 The recommended ways to run Wooey are:
 
+Through two separate processes
+------------------------------
+
+You can run Wooey by calling two commands (you will need a
+separate process for each):
+
+::
+
+    python manage.py celery worker -c 1 --beat -l info
+    python manage.py runserver
+
+On Windows, the ``--beat`` option might not be supported.
+
 Through a Procfile
 ------------------
 
-The simplest way to run Wooey is to use a Procfile with
+The best way to run Wooey is to use a Procfile with
```

Comment:
I agree about 'best', but I don't think that using a Procfile is the 'simplest' - the simplest is using two separate processes. How about 'recommended'?